Test/metatests/EarlyExitForUnsupportedFeatures.dfy.oldway.expect

```diff
@@ -0,0 +1,3 @@
+
+Dafny program verifier did not attempt verification
+EarlyExitForUnsupportedFeatures.dfy(6,8): Error: Feature not supported for this compilation target: Non-native numeric newtypes (non-native newtype positive)
```

Comment:
Just for the record, I'm not super proud of using `:` no less than 3 times in a single "sentence", but I can live with it as I can't think of a better way to cram all this information into a single error message. :)